piton/tools/src/proto/vivado/gen_project.tcl

```diff
@@ -177,7 +177,11 @@ set fileset_obj [get_filesets constrs_1]
 
 # Add/Import constrs file and set constrs file properties
 set constraints_file "${BOARD_DIR}/constraints.xdc"
+set board_constraints_file "${BOARD_DIR}/SUME_Master.xdc"
```

Comment:
I don't know about these changes in gen_project.tcl. Can we get them to be SUME-only somehow?